admin-app/dictionary/views.py

```
@@ -0,0 +1,150 @@
+    else:
+        word = Word.objects.order_by('?').first()
+    return JsonResponse(word.to_dict() if word else None)
+
+@require_GET
+def get_tags(request):
+    tags = Tag.objects.all()
+    tag_list = [{'name': tag.name, 'display_name': tag.display_name or tag.name} for tag in tags]
+    return JsonResponse(tag_list, safe=False)
+
+@require_GET    
+def word_detail(request, pk):
```

Ruff (0.15.2)

[warning] Unused function argument: `request`

(ARG001)